helm/microgateway/README.md

```diff
@@ -1,145 +1,293 @@
+
+| `wso2.deployment.wso2microgw.resources.requests.memory` | The minimum amount of memory that should be allocated for a Pod | 1Gi |
+
+| `wso2.deployment.wso2microgw.resources.requests.cpu` | The minimum amount of CPU that should be allocated for a Pod | 2000m |
+
+| `wso2.deployment.wso2microgw.resources.limits.memory` | The maximum amount of memory that should be allocated for a Pod | 2Gi |
+
+| `wso2.deployment.wso2microgw.resources.limits.cpu` | The maximum amount of CPU that should be allocated for a Pod | 2000m |
+
+| `wso2.deployment.wso2microgw.ingress.hostname` | Hostname for Microgateway | am.wso2.com |
+
+| `wso2.deployment.wso2microgw.ingress.annotations` | Ingress resource annotations for Microgateway  | - |
+
+| `wso2.deployment.wso2microgw.config` | Custom deployment configuration file for Microgateway  | - |
+
+  
 
 **Note**: The above mentioned default, minimum resource amounts for running WSO2 API Microgateway are based on its [official documentation](https://docs.wso2.com/display/MG301/Installation+Prerequisites#InstallationPrerequisites-MicrogatewayRuntime).
 
+  
+
 ##### 3. Deploy WSO2 Identity server.
 
+  
+
 ```
+
 helm install --dep-up --name <RELEASE_NAME> <HELM_HOME> --namespace <NAMESPACE>
+
 ```
 
+  
+
 `NAMESPACE` should be the Kubernetes Namespace in which the resources are deployed
 
+  
+
 ##### 4. Access Management Console.
 
-Default deployment will expose `<RELEASE_NAME>` host (to expose Administrative services and Management Console).
+  
+
+Default deployment will expose `<HOSTNAME>` host (to expose Administrative services and Management Console).
+
+  
 
 To access the console in the environment,
 
+  
+
 a. Obtain the external IP (`EXTERNAL-IP`) of the Ingress resources by listing down the Kubernetes Ingresses.
 
+  
+
 ```
+
 kubectl get ing -n <NAMESPACE>
+
 ```
 
+  
+
 ```
-NAME                       HOSTS                ADDRESS        PORTS     AGE
-wso2micro-gw-ingress       <RELEASE_NAME>       <EXTERNAL-IP>  80, 443   3m
+
+NAME HOSTS ADDRESS PORTS AGE
+
+wso2micro-gw-ingress <HOSTNAME> <EXTERNAL-IP> 80, 443 3m
+
 ```
 
+  
+
 b. Add the above host as an entry in /etc/hosts file as follows:
 
+  
+
 ```
-<EXTERNAL-IP>	<RELEASE_NAME>
+
+<EXTERNAL-IP> <HOSTNAME>
+
 ```
 
+  
+
 ## Enabling Centralized Logging
 
-Centralized logging with Logstash and Elasticsearch is disabled by default. However, if it is required to be enabled, 
+  
+
+Centralized logging with Logstash and Elasticsearch is disabled by default. However, if it is required to be enabled,
+
 the following steps should be followed.
 
+  
+
 1. Set `centralizedLogging.enabled` to `true` in the [values.yaml](values.yaml) file.
+
 2. Add elasticsearch Helm repository to download sub-charts required for Centralized logging.
+
 ```
+
 helm repo add elasticsearch https://helm.elastic.co
+
 ```
+
 3. Create a requirements.yaml at <HELM_HOME> and add the following dependencies in the file.
+
 ```
+
 dependencies:
```

Comment:
@chathuraswso2 did you test the chart with the WSO2 Kubernetes Pipeline? What is the version which you used?

Reply:
Didn't check with any Kubernetes Pipeline.

Reply:
@ThilinaManamgoda shouldn't we test this chart with the WSO2 Kubernetes Pipeline and update the necessary sample repositories? Please correct me if I am wrong.